media-libs/opencv/opencv4-4.4.0.recipe

```diff
@@ -0,0 +1,277 @@
+	devel:libopencv_features2d$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_flann$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_gapi$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_highgui$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_imgcodecs$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_imgproc$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_ml$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_objdetect$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_photo$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_stitching$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_video$secondaryArchSuffix = $libVersionCompat
+	devel:libopencv_videoio$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	opencv4$secondaryArchSuffix == $portVersion base
+	"
+
+PROVIDES_tools="
+	cmd:opencv_annotation$commandSuffix = $portVersion
+	cmd:opencv_interactive_calibration$commandSuffix = $portVersion
+	cmd:opencv_version$commandSuffix = $portVersion
+	cmd:opencv_visualisation$commandSuffix = $portVersion
+	cmd:setup_vars_opencv4.sh$commandSuffix = $portVersion
+	"
+REQUIRES_tools="
+	haiku$secondaryArchSuffix
+	opencv4$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:cblas$secondaryArchSuffix
+	devel:eigen$secondaryArchSuffix
+	devel:libavcodec$secondaryArchSuffix
+	devel:libavformat$secondaryArchSuffix
+#	devel:libavresample$secondaryArchSuffix
+	devel:libavutil$secondaryArchSuffix
+#	devel:libcaffe$secondaryArchSuffix
+	devel:libexif$secondaryArchSuffix
+	devel:libfreetype$secondaryArchSuffix
+	devel:libgdal$secondaryArchSuffix
+#	devel:libgflags$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libglib_2.0$secondaryArchSuffix
+	devel:libglog$secondaryArchSuffix
+	devel:libGLU$secondaryArchSuffix
+	devel:libgphoto2$secondaryArchSuffix
+	devel:libgstapp_1.0$secondaryArchSuffix
+	devel:libgstreamer_1.0$secondaryArchSuffix
+	devel:libgstvideo_1.0$secondaryArchSuffix
+	devel:libharfbuzz$secondaryArchSuffix
+	devel:libhdf5$secondaryArchSuffix
+#	devel:libilmimf$secondaryArchSuffix #openexr
+	devel:libintl$secondaryArchSuffix
+	devel:libjasper$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	devel:liblapack$secondaryArchSuffix
+	devel:libopenblas$secondaryArchSuffix
+	devel:libopencl_headers$secondaryArchSuffix
+	devel:liborc_0.4$secondaryArchSuffix
+	devel:libpng16$secondaryArchSuffix
+#	devel:libprotobuf$secondaryArchSuffix
+	devel:libQt5Concurrent$secondaryArchSuffix
+	devel:libQt5Core$secondaryArchSuffix
+	devel:libQt5Gui$secondaryArchSuffix
+	devel:libQt5Test$secondaryArchSuffix
+	devel:libQt5Widgets$secondaryArchSuffix
+	devel:libswscale$secondaryArchSuffix
+#	devel:libtesseract$secondaryArchSuffix
+	devel:libtiff$secondaryArchSuffix
+	devel:libwebp$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+#	cmd:ant
+	cmd:cmake
+	cmd:doxygen
+	cmd:f2py2
```

Comment:
Recipe for numpy_python-1.16.5.recipe is disabled for secondary architecture, thus only available at 64 bit (or with a local build